pliant/README.md

```diff
@@ -56,6 +57,7 @@ In this example, the workflow is called "RestartHost" and triggers a host restar
 `org`
 `id`
 
+We also have additional output variables (`host`, `meta`, and `ip`) we initialized. Our workflow will assign these output variables and output their resulting values on completion. We may also specify variables which are neither input nor output variables to use interally within our workflow's logic.
```

Comment:
```suggestion
There are also have additional output variables (`host`, `meta`, and `ip`) that are initialized. The workflow will assign these output variables and output the resulting values upon completion. It may also specify variables which are neither input nor output variables to use internally within the workflow's logic.
```